### ISSUE TITLE
Display Miners/Pumpjacks seems to be doing nothing

### ISSUE THREAD

**Issue by Chakani42** [Rybadour/Foreman#23]
It seems that "Display Assemblers" controls the display of the needed Miners for the resource outputs, as opposed to what I would assume is expected behavior of it displaying with "Display Miners/Pumpjacks".

**Comment by Rybadour** [comment]
Agreed this is a bug. "Display Miners/Pumpjacks" should be the only thing that toggles if Miners appear on "infinite" nodes for things such as ore or water.
